File: backendworker/NHSOnline.Backend.Worker/GpSystems/Suppliers/Vision/Models/Appointments/Owner.cs
using System.Xml.Serialization;

namespace NHSOnline.Backend.Worker.GpSystems.Suppliers.Vision.Models.Appointments
{
    public class Owner
    {

        [XmlAttribute(AttributeName = "id")]
        public string Id { get; set; }

        [XmlElement(ElementName = "name")]
        public string Name { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/vision/models/appointments/Owner.kt
package mocking.vision.models.appointments

import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlElement

data class Owner(
        @XmlAttribute(name = "id") private var id: Int?= null,
        @XmlElement(namespace = "urn:vision")
        var name: String
)
